src/screen.ts

```diff
@@ -42,6 +42,7 @@ const logTestingPlaygroundURL = (element = getDocument().body) => {
   console.log(
     `Open this URL in your browser\n\n${getPlaygroundUrl(element.innerHTML)}`,
   )
+  return getPlaygroundUrl(element.innerHTML)
```

Comment:
Small nit, can we save the `getPlaygroundUrl in a const instead of calling the function twice? Not sure what the performance aspects for it, but better safe than sorry :)